### ISSUE TITLE
Audio playback issue in Windows binary.

### ISSUE THREAD

**Issue by fuzzy-man** [marlam/bino#14]
Hello.

Sound does not work properly when playing video in Windows. Only the first fragment is played and then silence. Tried on different videos.

I think the problem is with audio tracks encoded in ac3. All the videos I've tried have multiple audio tracks.

**Comment by marlam** [comment]
Hi, I am currently investigating a problem with the Windows version of Bino that might be related to this. Do you have an example file where this happens?

**Comment by marlam** [comment]
Alternatively, can you check if this new version fixes the problem?
https://bino3d.org/releases/bino-1.6.8b-w64.exe

This fixes a problem with thread handling on Windows which might be related to your problem.

**Comment by fuzzy-man** [comment]
Tried both versions, but result seems the same.

I'm trying to watch some 3d movies. Also tried 2d.

Here screenshots of metadata of files that run with audio and without audio.

Video without audio:
![no_audio](https://user-images.githubusercontent.com/109215218/181631277-d729f2d3-2f7a-4a94-a87b-0f3bc57d1287.png)

Video with audio:
![audio_ok](https://user-images.githubusercontent.com/109215218/181631292-9615d5a2-9c77-4538-a6fc-36bcf176f6c3.png)

Looks like I have video files with ac3 codec which run good. The other video has resolution of 1080p and is TaB 3d video.

I've noticed that videos that don't play sound have subtitles which are set off by default. I also tried to switch them, but the application freezes for a long time.

My computer on which I want to run Bino: AMD Phenom II x4, 16 Gb RAM, Nvidia GeForce GTX 560, Windows 10 64 bit. CPU load is low during playback.

I also tried on more modern notebook with AMD Ryzen 5500U CPU and integrated video card, On Windows 11 it runs the same videos with sound but I needed to do pause-play video after full screen or vice versa or when changing audio track. There is also Fedora Workstation 36 on this devise and it runs good in X session (version is 1.6.8 from the rpm fusion repo).
